sympy/solvers/tests/test_solvers.py

```diff
@@ -1659,7 +1659,7 @@ def test_issue_5849():
 
     v = I1, I4, Q2, Q4, dI1, dI4, dQ2, dQ4
     assert solve(e, *v, manual=True, check=False, dict=True) == ans
-    assert solve(e, *v, manual=True, check=False) == ans[0]
+    assert solve(e, *v, manual=True, check=False) == ans
```

Comment:
`manual=True` does not use linear tracking; if `manual=True` and there is one solution it could be unpacked, but then this would potentially unpack nonlinear results leading to a disrepancy with the non-manual handling. So to keep it simple, just don't unpack manual results.

Reply:
Or (as I think it will be here) leave as tuple when symbols are ordered.